remote-web/src/styles.css

```
@@ -0,0 +1,954 @@
+  --muted-2: #667382;
+  --line: #28323d;
+  --line-strong: #3e4b58;
+  --green: #50e39f;
+  --blue: #63b7ff;
+  --yellow: #f0c66a;
+  --red: #ff756f;
+  --purple: #d58cff;
+  --shadow: 0 24px 84px rgba(0, 0, 0, 0.48);
+  --mono: "SF Mono", Menlo, Monaco, Consolas, monospace;
+  --remote-visual-height: 100dvh;
+  font-family: -apple-system, BlinkMacSystemFont, "SF Pro Text", "Segoe UI", sans-serif;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)